### ISSUE TITLE
Map run button to the start command

### ISSUE THREAD

**Issue by jtpio** [jupyterlab/debugger#81]
Now that we have the `start` and `stop` commands, they can be triggered when clicking on the start / continue and stop buttons.

**Comment by jtpio** [comment]
The run button here refers to the one in the Callstack panel:

![image](https://user-images.githubusercontent.com/591645/67842427-73053580-fafa-11e9-98ae-a89e2647f3ff.png)

However it is similar to the "Run Cell". So maybe it can be enabled only when a "Continue" action is possible (thread stopped).

**Comment by JohanMabille** [comment]
Since we decided to automagically start the debugger when it's posible and never stop it, I think that the `start` command should not be mapped to the continue button. The `stop` button should continue the execution until the entire cell has been executed. This can be done thanks to an additional method in the service that automagically sends continue when the code hits a breakpoint.

**Comment by jtpio** [comment]
Indeed, the title of the PR needs an update.

**Comment by jtpio** [comment]
Also, that means passing down functions that will perform these actions down to the `CallStack` panel. The `CallStack` panel doesn't have access to the `DebugService` and the `CommandRegistry`.

**Comment by JohanMabille** [comment]
The `DebuggerPanel` constructor is passed the `service` object, it can propagate it to its subpanels.

**Comment by jtpio** [comment]
Indeed, related to the recent changes we can now pass the `service` object to the callstack panel.
